assets/js/src/post_editor_block/form_block.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const wp = window.wp;
+const { registerBlockType } = wp.blocks;
+
+registerBlockType('mailpoet/form-block', {
+  title: 'Example: Basic (esnext)',
+  icon: 'universal-access-alt',
+  category: 'widgets',
+  example: {},
+  edit() {
+    return (
+      <div className="mailpoet-block-div">Hello World, step 1 (from the editor).</div>
```

Comment:
Is the `-div` suffix something we use (or Gutenberg) in other places? I'm wondering whether that suffix is necessary (or whether it should be something rather semantical such as `-wrapper`).

Reply:
No, this was a temporary code form a tutorial.